app/assets/javascripts/repository_navigation.js

```diff
@@ -68,6 +79,33 @@
         sendForm();
       }
     });
+
+
+    /*
+    Close checkout instructions
+    */
+    var checkout = $('#repository--checkout-instructions'),
+      toggle = $('#repository--checkout-instructions-toggle');
```

Comment:
I think there are styling rules saying that `toggle` and `checkout` need to be aligned?

(from my point of view you could as well spam `var` keywords, but I think nobody else would like that ^^